sbc_compassion/tests/test_tools.py

```diff
@@ -121,7 +121,8 @@ def _read_img(self, path):
 
     def _pattern_recognition(self, path):
         img = self._read_img(path)
-        return tools.patternrecognition.find_template(img, self.templates, 0.3)
+        return tools.patternrecognition.find_template(
+            img, self.templates, 0.3)[:2]
```

Comment:
I'm not fan of this, as it seems a bit "magic" to cut the result. By reading the code we don't understand what's going on. Can you return all data and just ignore what is not needed in the tests ?